ClientApp/src/Pages/SubmitSighting/SubmitSighting.tsx

```diff
@@ -52,6 +64,7 @@ export function SubmitSightingForm(): JSX.Element {
           className="form-input"
           value={quantity}
           onChange={(event) => setQuantity(event.target.value)}
+          type="number"
```

Comment:
This is getting changed in emily and roshni's PR to be a string in the backend to accept results such as 20+, it might be worth discussing with them if you think we should support that kind of input in our website too

Reply:
All references to quantity have now been changed to strings